docs/ref/contrib/messages.txt

```diff
@@ -258,6 +258,11 @@ The ``Message`` class
       the lowercase version of the name of the associated constant, but this
       can be changed if you need by using the :setting:`MESSAGE_TAGS` setting.
 
+    * ``extra_kwargs``: This variable accepts all further keyword arguments
```

Comment:
I'm not entirely happy with this, but believe it should be documented. Perhaps as a new section titled something like: `Custom Storage & Message classes`

Reply:
Also needs a `..versionchanged` note and reference in the release notes

Good point, we have a bit on BaseStorage and how you theoretically can make a custom storage (https://docs.djangoproject.com/en/5.0/ref/contrib/messages/#django.contrib.messages.storage.base.BaseStorage) adding something in a section there demonstrating the value of this would really help

If you make a really simple example, that might also be useful for writing a test